tile_server/README.md

```diff
@@ -2,42 +2,44 @@
 
 This code provides a Go server to handle
 [Tile Map Service](https://en.wikipedia.org/wiki/Tile_Map_Service) requests. It
-is able to respond with [GeoJSON](https://geojson.org) or image tiles, either of
+is able to respond with [GeoJSON](https://geojson.org), image tiles or
+[Mapbox Vector Tile](https://github.com/mapbox/vector-tile-spec), any of
```

Comment:
"Mapbox Vector Tiles" (plural)

Also can you include the version (I guess it's 2.1)?